extension/js/common/api/key-server/keys.openpgp.org.ts

```diff
@@ -0,0 +1,46 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
```

Comment:
This is really a nitpick but it doesn't seem common to use dots here, the style we use is `ClassName` that becomes `class-name.ts`

Reply:
ok